### PR TITLE
added meta-data and json-ld structured data

### PR DESCRIPTION
**Pull request by ganeshrevadi**
# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes (https://github.com/keploy/keploy/issues/803)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -30,14 +30,40 @@ module.exports = {
...
-        description: '',
+        description: "",
       },
       {
-        name: 'x-default',
-        content: 'en-us',
+        name: "x-default",
+        content: "en-us",
+      },
+      {name: "keywords", content: "cooking, blog"},
```
Comment (nehagup) [inline docusaurus.config.js:42]: keywords are not right and are directly copied from the sample given.

File: docusaurus.config.js [hunk 2]
```diff
@@ -30,14 +30,40 @@ module.exports = {
...
+      {name: "twitter:card", content: "summary_large_image"},
+    ],
+    headTags: [
+      // Declare a <link> preconnect tag
+      {
+        tagName: "link",
+        attributes: {
+          rel: "preconnect",
+          href: "https://example.com",
```
Comment (nehagup) [inline docusaurus.config.js:51]: Copy-pasted!

**Review by nehagup — COMMENTED** [review]
@ganeshrevadi Thanks for the PR. 

There are unnecessary linting changes like replacing '' with "" that are not relevant to this PR.

**Comment by ganeshrevadi** [comment]
> @ganeshrevadi Thanks for the PR.
> 
> There are unnecessary linting changes like replacing '' with "" that are not relevant to this PR.

I have made the changes !

**Review by nehagup — CHANGES_REQUESTED** [review]
@ganeshrevadi there are still listing changes like - "" replaced with '' which are not related to this PR. Can you please revert those back and make the necessary changes only?

**Comment by ganeshrevadi** [comment]
> @ganeshrevadi there are still listing changes like - "" replaced with '' which are not related to this PR. Can you please revert those back and make the necessary changes only?
Hey @nehagup ! I have created a new PR for the following changes 
(https://github.com/keploy/docs/pull/203)

**Comment by ganeshrevadi** [comment]
Can this PR be closed?

**Comment by Sonichigo** [comment]
Closing this PR since new changes are in PR #203

**Pull request closed without merging**